sample-app/screens/Home/HomeData.tsx

```diff
@@ -29,16 +54,8 @@ export const HomeScreenData = {
                     "subtitle": "to be stored for future transactions",
                     "type": HomeListType.SaveCard
                 } as HomeListItem,
-                {
-                    "title": "Apple Pay payment",
-                    "subtitle": "with a wallet card",
-                    "type": HomeListType.ApplePay
-                } as HomeListItem,
-                {
-                    "title": "Apple Pay pre-auth",
-                    "subtitle": "with a wallet card",
-                    "type": HomeListType.ApplePreAuth
-                } as HomeListItem,
+                Platform.OS == "ios" ? applePayment : googlePayment,
```

Comment:
can we extract the `Platform.OS == "ios"` to a separate helper function, to avoid repetition?
something like `const isIOS = () => Platform.OS == "ios"`

Reply:
There is utils.tsx file with those methods

Reply:
Changed. Resolve if fixed.